src/Zed/ruleset.xml

```diff
@@ -12,117 +12,99 @@
 
     <!-- BUSINESS FACADE RULES -->
     <rule
-        name="AllMethodsPublicInFacadeRule"
-        message="Zed Business Layer - Facade: {0}"
-        class="ArchitectureSniffer\Zed\Business\Facade\AllMethodsPublicInFacadeRule">
+            name="AllMethodsPublicInFacadeRule"
+            message="Zed Business Layer - Facade: {0}"
+            class="ArchitectureSniffer\Zed\Business\Facade\AllMethodsPublicInFacadeRule">
 
         <priority>1</priority>
     </rule>
 
     <rule
-        name="ArgumentsFacadeRule"
-        message="Zed Business Layer - Facade: {0}"
-        class="ArchitectureSniffer\Zed\Business\Facade\ArgumentsFacadeRule">
+            name="ArgumentsFacadeRule"
+            message="Zed Business Layer - Facade: {0}"
+            class="ArchitectureSniffer\Zed\Business\Facade\ArgumentsFacadeRule">
 
-        <priority>2</priority>
+        <priority>3</priority>
```

Comment:
Yes: https://github.com/spryker/architecture-sniffer#levels
and also soon in the high level wrapper in Development module.